include/ultra64.h

```diff
@@ -1,13 +1,16 @@
 #ifndef ULTRA64_H
 #define ULTRA64_H
 
-#include "stddef.h"
-#include "stdbool.h"
-#include "math.h"
+#include "ultra64/types.h"
 #include "unk.h"
 
-#include "ultra64/types.h"
-#include "ultra64/interrupt.h"
+#include "libc/stdarg.h"
+#include "libc/stdbool.h"
+#include "libc/stddef.h"
+#include "libc/stdlib.h"
+#include "libc/math.h"
```

Comment:
Just a thought and we don't need to do anything about this right now, but does it make sense to have these includes in `ultra64.h`? Same question for `unk.h` tho that one is probably easier to answer.

Reply:
The libc headers are needed in libultra, and of course in other parts of the codebase. I think it's appropriate currently to have them in `ultra64.h` but this may change depending on how libultra headers go. It also depends how we want to include libultra headers in non-libultra files, whether we just include `ultra64.h` or if we want to be more specific with our libultra includes. If the former, I think the libc headers are good to stay here.